packages/app-page-builder-elements/src/renderers/form/types.ts

```diff
@@ -77,8 +84,12 @@ export interface ErrorResponse {
 export type FormLayoutComponentProps<T = any> = {
     getFieldById: Function;
     getFieldByFieldId: Function;
-    getFields: () => FormRenderComponentDataField[][];
+    getFields: (stepIndex: number) => FormRenderComponentDataField[][];
     getDefaultValues: () => { [key: string]: any };
+    goToPreviousStep: () => void;
+    validateCurrentStepFields: (form: FormAPI) => void;
```

Comment:
There's a possibility we'll be able to get rid of this method. We'll see.

Reply:
@adrians5j Hi, could you please check pr

Reply:
@adrians5j sorry for bothering, but could you please check this pr again 🙏🏻